lib/test.js

```diff
@@ -21,7 +21,7 @@ var Test = function(options) {
   }
   
   this.web3 = new Web3();
-  this.web3.setProvider(this.sim.provider());
+  this.web3.setProvider(this.sim.provider(options));
```

Comment:
The ```options``` variable is actually not meant to be used by the provider (it's for the Test object itself), do you have something in mind by being able to pass options to the testrpc provider?